### DISCUSSION TITLE
Bad Start mod

### DISCUSSION THREAD

**Discussion by liljaba1337** [ppy/osu#27860] [Ideas]
If you get 100/50/miss in first N seconds, you fail (and restart, optional). N can be adjusted in the mod settings. So if you start badly, it restarts for you.

**Comment by Stoppedpuma** [comment]
I don't think an entire mod is needed for this, you can set this option as a key near your input keys to restart quickly.

![1713130896](https://github.com/ppy/osu/assets/58333920/7cbfd92e-0303-4870-a852-88d3ca86d6e3)

**Comment by shiumano** [comment]
If you want perfection that badly, why not use the Accuracy Challenge?

With some light math, you can set it to die if you get 100s on how many notes.
